### ISSUE TITLE
Data uncertainty or model uncertainty?

### ISSUE THREAD

**Issue by ZhouCX117** [gigantenbein/UNet-Zoo#7]
Thanks for your excellent code!
Could you please tell me what kind of uncertainty the phiseg network considers? Data uncertainty or model uncertainty?

**Comment by gigantenbein** [comment]
Thank you!

PhiSeg was trained on datasets where multiple annotations for the same image were available. The images obtained by sampling from PhiSeg can then be considered as "likely" annotations of the image.  The GED and NCC scores calculated in [RevPhiSeg](https://arxiv.org/abs/2008.06999) , [PhiSeg](https://arxiv.org/abs/1906.04045) and [ProbabilisticUnet](https://arxiv.org/abs/1806.05034v4) thus are a way of quantifying data uncertainty.

To answer your question: you can say that it only considers data uncertainty of the images fed to the network. 

I hope that helps.

**Comment by ZhouCX117** [comment]
@gigantenbein Thanks for your reply! You are so nice~
Could I ask one more question?
Are there any other methods to measure model uncertainty besides Bayesian neural network, ensembles, and MC dropout?
